### PR TITLE
Repair quickstart - Changing num threads to 9

### PR DESCRIPTION
**Pull request by maver1ck**
Repair for issue:
https://github.com/druid-io/druid/issues/3491

### CODE REVIEW

File: examples/conf-quickstart/druid/broker/runtime.properties [hunk 1]
```diff
@@ -3,7 +3,7 @@ druid.port=8082
 
 # HTTP server threads
 druid.broker.http.numConnections=5
-druid.server.http.numThreads=8
```
Comment (b-slim) [inline examples/conf-quickstart/druid/broker/runtime.properties]: the exception seems to be related to tranquility, not sure if this is needed here as well ?

Comment (maver1ck) [inline examples/conf-quickstart/druid/broker/runtime.properties]: Yep. That was sample exception.
There are same exceptions when running broker etc.

**Comment by b-slim** [comment]
👍

**Comment by maver1ck** [comment]
Ping ?

**Comment by dclim** [comment]
This seems to be happening because of the following lines in org.eclipse.jetty.server.AbstractConnector:

```
        int cores = Runtime.getRuntime().availableProcessors();
        if (acceptors < 0)
            acceptors=Math.max(1, Math.min(4,cores/8));
```
We use the default number of acceptors so it gets set to 4 if machine has >= 32 cores.

@maver1ck could you fill out the CLA posted here: http://druid.io/community/cla.html ? Once that's done, we can accept the PR.

**Comment by maver1ck** [comment]
@dclim 
Done

**Comment by dclim** [comment]
thanks! 👍

**Pull request merged**